db/schema.rb

```diff
@@ -55,16 +55,14 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "instance_admin", default: false
+    t.string "stripe_customer_id"
     t.index ["email"], name: "index_registered_users_on_email", unique: true
     t.index ["reset_password_token"], name: "index_registered_users_on_reset_password_token", unique: true
   end
 
   create_table "stripe_connections", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "owner_id"
     t.string "stripe_account_id"
-    t.string "business_name"
-    t.string "display_name"
-    t.string "statement_descriptor"
```

Comment:
I don't know why this was removed? Probably because I rebased wrong or something?